reV/supply_curve/supply_curve.py

```diff
@@ -399,8 +409,10 @@ def _parse_trans_table(sc_points, trans_table, fcr, trans_costs=None,
         rename = {p: t for p, t in zip(point_merge_cols, table_merge_cols)}
         sc_cap = sc_cap.rename(columns=rename)
 
+        logger.debug('Merging SC table and Trans Table on columns: {}'
+                     .format(table_merge_cols))
         trans_table = trans_table.merge(sc_cap, on=table_merge_cols,
```

Comment:
This method has become pretty long and complex. What do you think about pulling out this middle bit that does the merge between sc_points and the trans_table and putting that into a new staticmethod?

Reply:
Agreed! Broke out all methods into single purpose static methods (merge, check, lcot). This is the problem with multi-purpose "parse()" methods.... They become do-everything methods. Gotta be cautious of that in the future.